src/KDSoapClient/KDSoapClientInterface.h

```diff
@@ -346,9 +346,46 @@ class KDSOAP_EXPORT KDSoapClientInterface
       */
     void setTimeout(int msecs);
 
+    /**
+     * @brief setSendHTTPActionParameter
+     * @param sendHTTPAction
+     * This option can be used to enable/disable the sending of the SOAP action parameter
+     * in the Content-Type HTTP header (in the case of SOAP 1.2 requests).
+     * It might be necessary to disable it for certain  SOAP server implementations
+     * in the case if the action is passed in the SOAP envelope's header with WS-Addressing.
+     */
```

Comment:
Missing `@since 1.9` for all 4 new methods.

Reply:
I have changed the placeholders from @param format to \param to be consistent with the previous method comments.